Add short post "Mountain Trip '22" (#74)
alasco-tech/alasco-tech.github.io: _posts/2022-03-31-alasco-mountain-trip-22.md

---
author: engineering-team
date: 2022-03-31
layout: post
title: Mountain Trip '22
subtitle: We recently got back from our Alasco mountain trip.
thumb: mountain_trip_mountain.jpg
teaseralt: Mountains with snow.
description: We recently got back from our Alasco mountain trip.
---

We recently got back from our Alasco mountain trip that took place on the 19th and 20th of March.

![Alascians having lunch on a mountain](/assets/images/mountain_trip_lunch.jpg "Alascians having lunch on a mountain")

> Team events are an important detail of our organisation. They happen at team level, at function & department level and at company level and Alasco goes far to ensure that they are an exciting and enjoyable experience for everyone.

Reviewer: Would use "and" instead of ampersands in paragraphs.

> Team events are an important detail of our organisation. They happen at team level, at function & department level and at company level and Alasco goes far to ensure that they are an exciting and enjoyable experience for everyone.

Reviewer 2: ```suggestion
Team events are an important detail at Alasco, and we go as far as to ensure that they are exciting and enjoyable experiences for everyone. Such team events happen at both team, department and company levels.
```

> The modern process of building software requires close collaboration of many stakeholders — and that collaboration benefits from inter-team connections and cross-pollination of ideas. While we follow processes in our day to day, good ideas also arise from unplanned interactions. With team events and activities, we are fostering a dense interaction and trust graph at Alasco.

Reviewer 2: NIT: This paragraph contains complex vocab compared to the one above it. Would recommend simplifying it to match the vibe. Otherwise very good :)

Reviewer 2: ^ In regards to the second paragraph
